benchopt/runner.py

```diff
@@ -48,6 +49,12 @@ def colorify(message, color=BLUE):
 @mem.cache
 def run_repetition(objective, solver_class, solver_parameters, meta, sample):
 
+    # check if the module caught a failed import
+    module = importlib.import_module(solver_class.__module__)
+    if module.solver_import.failed_import:
+        raise ImportError(
+            f"Failure during import in {solver_class.__module__}.")
+
```

Comment:
Why do you add this? It seems redundant with l.122 but I am maybe missing how you could end up in the situation where `run_repetition` is called while `run_one_solver` is still called.

Reply:
Yes, I first added it to `run_one_solver` to raise an ImportError, then I went further to only print "failed" in `run_repetition`, but I figured I could leave it in `run_one_solver` in the case that it is called from another function.

Reply:
Ok it is fine by me.